pkg/releases/consumer_test.go

```diff
@@ -0,0 +1,138 @@
+package releases
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/atlassian/voyager/pkg/releases/deployinator/models"
+	"github.com/pact-foundation/pact-go/dsl"
+)
+
+/**
+This test tests the consumer interactions surrounding the resolve endpoints between
+voyager (consumer) and trebuchet (provider).
+
+To validate the pact against the current provider specs locally, use the contract testing cli found here:
+https://hello.atlassian.net/wiki/spaces/TESTA/pages/128609304/Contract+Testing+Getting+Started+Guide+for+Consumers
+ */
+func TestConsumerPact(t *testing.T) {
+
+	// Initialize the pact
+	pact := &dsl.Pact{
+		Consumer: "voyager",
+		Provider: "deployinator-trebuchet",
+		Host:     "localhost",
+		PactDir:  "generated/pacts",
+	}
+
+	defer pact.Teardown()
+
+	// Define the interactions in resolve and batchResolve separately, with the same pact
+
+	ResolveEndpointTest(pact)
```

Comment:
We wanted these two to generate a single pact contract for us (under the same pact struct), so if we split it into two, it would generate 2 seperate contracts.